proposal.md

```diff
@@ -129,13 +129,17 @@ CONSIDER: pkgdown, learnr
 
 ### Package metadata and information files
 
-`tools::CRAN_package_db` returns a data frame with character columns containing most `DESCRIPTION` metadata for the current packages in the CRAN package repository.
+`tools::CRAN_package_db()` returns a data frame with character columns containing most `DESCRIPTION` metadata for the current packages in the CRAN package repository.
```

Comment:
There have been many functions added about packages on CRAN on r-devel. See: https://developer.r-project.org/blosxom.cgi/R-devel/NEWS/2024/08/22#n2024-08-22 they can be useful to avoid topic duplication or even to find packages. Perhaps they are worth a mention.

Reply:
According to the [R developement guidelines](https://developer.r-project.org/devel-guidelines.txt) new functions will not be added until the next major/minor release of R next April. Hoping we get this task view finished before then, I will add the idea of incorporating these new functions as an issue.